cookbook/static-route-controller.rst

```diff
@@ -0,0 +1,66 @@
+Create a static route controller
```

Comment:
not sure if the term `static route controller` is used in the symfony docs. also you dont need to do this if you dont want to render your page template right?

Reply:
maybe `Render page template from controller` / `Render base template from controller` or `Render page template from custom route`?

Reply:
Should we go with something like "How to get sulu twig template attributes" `sulu-twig-attributes.rst`.

Reply:
yes, thats good too 🙂
maybe `Provide Sulu twig template attributes in your controller`?